Sources/GRPCCodeGen/Internal/Translator/TypealiasTranslator.swift

```diff
@@ -270,4 +323,36 @@ extension TypealiasTranslator {
       )
     )
   }
+
+  private func makeServiceDescriptorExtension(
+    for service: CodeGenerationRequest.ServiceDescriptor
+  ) -> Declaration {
+    let serviceIdentifier =
+      service.namespace.generatedLowerCase.isEmpty
+      ? service.name.generatedLowerCase
+      : "\(service.namespace.generatedLowerCase)_\(service.name.generatedUpperCase)"
```

Comment:
Use the ternary operator if the condition and branches are short (e.g. `let foo = x > 10 ? bar : baz`) otherwise they're just less readable. 

If it can't fit on one line then just use an `if`/`else` – it will be easier to read (this also _doesn't_ mean that if it fits on one line then it's always acceptable to use the ternary operator).

Reply:
In this instance you probably want to split the identifier into two parts: a prefix and the service name:

```swift
let prefix = service.namespace.isEmpty ? "" : service.namespace + "_"
let serviceIdentifier = prefix + service.name.generatedUpperCase
```